### ISSUE TITLE
Interpreting 16S metagenomic data without clustering to achieve sub-OTU resolution.

### ISSUE THREAD

**Issue by pschloss** [SchlossLab/march_madness_2015#55]
Interpreting 16S metagenomic data without clustering to achieve sub-OTU resolution.
Tikhonov M, Leach RW, Wingreen NS.
ISME J. 2015 Jan;9(1):68-80. doi: 10.1038/ismej.2014.117. Epub 2014 Jul 11.
http://www.ncbi.nlm.nih.gov/pubmed/25012900

**Comment by nbaxter13** [comment]
Tikhonov et al. present an alternative approach to analyzing 16S data that doesn't involve clustering into OTUs.  The authors point out that clustering into OTUs, especially at the 97% cutoff, can make it impossible to resolve strain-level dynamics within a community, as some OTUs may contain multiple ecologically distinct populations.  They remind us that sequence similarity does not necessary imply ecological similarity, which is a common criticism of 16S rRNA sequencing.  Essentially their solution is to use unique sequences rather than OTUs. They used a longitudinal dataset of tongue microbiome samples collected from two cohabitating individuals(V4 region, Illumina) to demonstrate the benefits of their approach.  

They showed that sequence similarity is very weakly correlated with how strongly two sequences covary(what they call "dynamic similarity"), and only at high sequence similarities.  Supporting the idea that OTUs are bad.  They highlighted several examples where sequences within an OTU have very different temporal dynamics, suggesting they are distinct populations. They also showed that they could predict whether two nearly identical (99%) sequences came from the same population based on their dynamic similarity and the persistence of their difference in abundance. Finally they showed that using 100% identity better correlated the dynamics of populations between the two individuals. 

The authors point out that their technique on focusses on the most abundant members of the community, since it requires removing a lot of less abundant sequences for denoising.  They also note that its less useful "when studying coarse features of community composition and dynamics, e.g., comparing communities across habitats".  I think the method would work poorly for noisy datasets like large crossectional studies with human stool.  However, I think their arguments against OTUs are important to keep in mind.  If you're interested in a finer-scale study of the temporal dynamics of a community, then you should consider this approach. 

tl;dr - OTUs don't give good resolution. Consider using unique sequences for time-series data.

Overall: 6
Importance: 5
Innovation: 4
Who should read: @alyxwithay @mjenior @pschloss
Tags: 16S, OTUs, mothur
